README.md

```diff
@@ -193,18 +232,32 @@ In summary, having this composite key ensures it will scale well while also allo
 Though, the current schema has its limitations. There can't be 2 questions created at the same time because these questions would have the same primary key. To avoid that we could add another id (UUID) as a second clustering key like the following.
 
 ```
-CREATE TABLE qa.question (
-    date TEXT PARTITIONKEY,
-    created_at BIGINT CLUSTERINGKEY,
-    uuid BIGINT CLUSTERINGKEY,
-    title TEXT,
-    user TEXT,
-    context TEXT,
-    updated_at BIGINT,
-    number_of_answers INT,
-);
+{
+  "qa.question": {
+    "transaction": false,
+    "partition-key": [
+      "date"
+    ],
+    "clustering-key": [
+      "created_at",
+      "uuid"
+    ],
+    "columns": {
+      "date": "TEXT",
+      "created_at": "BIGINT",
+      "uuid": "BIGINT",
+      "title": "TEXT",
+      "user": "TEXT",
+      "context": "TEXT",
+      "updated_at": "BIGINT",
+      "number_of_answers": "INT"
+    }
+  }
+}
 ```
 
+If it's need to specify `NETWORK STRATEGY` for Cassandra and `BASE RESOURCE UNIT` for cosmos DB, please see [Schema Tool for Scalar DB](https://github.com/scalar-labs/scalardb/blob/master/tools/scalar-schema/README.md)
```

Comment:
```suggestion
You may want to specify `NETWORK STRATEGY` for Cassandra and `BASE RESOURCE UNIT` for Cosmos DB depending on your needs. Please see [Schema Tool for Scalar DB](https://github.com/scalar-labs/scalardb/blob/master/tools/scalar-schema/README.md)
```

Reply:
Sorry for the inconvenience.
The document updated accordingly.